src/utils/runWithTruffle.js

```diff
@@ -19,6 +23,17 @@ function initTruffle(network) {
   });
 }
 
-export default function runWithTruffle(script, network) {
+function compileWithTruffle() {
+  log.info("Compiling contracts with truffle")
```

Comment:
I think truffle already outputs this one for you

Reply:
I left that one in case truffle fails silently.